### DISCUSSION TITLE
Why not move the build instructions to a wiki?

### DISCUSSION THREAD

**Discussion by lucydodo** [sqlitebrowser/sqlitebrowser#3473] [General]
I was recently rewriting build instructions and had a question.
Currently, our build instructions are written in two places, each with different content. This can be confusing for users.
- [BUILDING.md in code-base](https://github.com/sqlitebrowser/sqlitebrowser/blob/master/BUILDING.md)
- [Wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki)

So, it would be better to merge them into one place, but I personally think a wiki would be a better than a code-base.
What do you guys think? 🤔

**Comment by scottfurry** [comment]
The `BUILDING.md` is a markdown file that people can immediately review, although it's references are dated (seriously, who still uses Qt5.7?). It could use some clean up. The file is available with cloning the repository.

The WIKI is the _canonical_ resource. People who would do packaging for different distributions (SUSE, Red Hat, Arch, et al) would want to look here for exact details along with any pitfalls/considerations.

However, each should refer to the other. The existence of these resources need to be current but separate as they serve different use cases.

My $/€/£/¥ 0.02 worth.

**Comment by justinclift** [comment]
Yeah, having stuff on the wiki is a good idea.  Like @scottfurry mentions though, having info for *that specific release* that's in the release itself is also useful.

One problem with wiki documentation is that it's *generally* targeted to the latest source code.  So, if some major change in build process happens then when the wiki docs get updated it won't be accurate for the older releases.

At least, not without having separate pages.  "For release 0.1.0, go HERE.  For release 0.5.0, go THERE, etc".

Wiki pages *are* much easier to update than creating PRs for documentation changes though. :smile:

**Comment by lucydodo** [comment]
So why not keep it simple in a markdown file and more detailed in a wiki? :)

**Reply by justinclift** [reply]
Sure, we can try it out. :smile:
